admin/listPurchases/listView.html

```diff
@@ -209,11 +209,23 @@
 
 function describeItem(item){
     if(item.id && item.count){
-        return `${item.id}: <b>${item.count} stk</b>`;
+        return `${badgeNameFromId(item.id)}: <b>${item.count} stk</b>`;
     }
     return item.description;
 }
 
+function badgeNameFromId(id){
+    const badgeIdNameMapping = {
+        {{#badgesForSale}}
+        "{{ &id }}" : "{{ &name }}",
+        {{/badgesForSale}}
+    };
+    if(!badgeIdNameMapping[id]){
+        return id;
+    }
+    return badgeIdNameMapping[id];
+}
```

Comment:
As we talked about, I don't like this approach as it pushes lots more data and work to the client, instead of minimizing client-side work by doing it server-side.

Reply:
You haven't responded to this.

I think this is fine to merge *for now* but I'd like this fixed in the future!